app/dashboard/views.py

```diff
@@ -2765,3 +2774,33 @@ def get_hackathons(request):
         'hackathons': events,
     }
     return TemplateResponse(request, 'dashboard/hackathons.html', params)
+
+@require_POST
+@login_required
+def upload_profile_banner(request):
+    """Handle Profile Banner Uploads"""
+
+    filename = request.FILES.get('banner')
+    print(filename)
```

Comment:
we should disable this (its a security concern).  if we don't then we should probably make sure it at least don't accept any viruses (like the last time we got a security vuln report from mubassir about this @danlipert )

Reply:
`print(filename)`?